test/libsolidity/StandardCompiler.cpp

```diff
@@ -397,15 +398,15 @@ BOOST_AUTO_TEST_CASE(basic_compilation)
 		"{\"begin\":0,\"end\":14,\"name\":\"PUSH\",\"value\":\"40\"},"
 		"{\"begin\":0,\"end\":14,\"name\":\"MSTORE\"},"
 		"{\"begin\":0,\"end\":14,\"name\":\"CALLVALUE\"},"
-		"{\"begin\":8,\"end\":17,\"name\":\"DUP1\"},"
```

Comment:
Why did this change?

Reply:
Still trying to figure it out

Reply:
@chriseth I still don't really know why this happened. Because in some places the code now uses `appendInlineAssembly` instead of just appending `revert(0, 0)`, `/* CODEGEN ... */` was added to some instructions. I suspect it's related to that. Any hints you can think of?

Reply:
ah yes! It might use the source locations of the inline assembly string.